src/schemas/ProgramDonorSummary/resolvers/summaryEntries.ts

```diff
@@ -62,13 +62,23 @@ const programDonorSummaryEntriesResolver: (
     });
   }
 
+  const queries : Query[] = [];
+  queries.push(esb.termsQuery(EsDonorDocumentField.programId, programShortName));
+
+  args.filters.map(filter => {
+    const field = filter.field;
+    if (field === EsDonorDocumentField.combinedDonorId && filter.values.length > 0) {
+      // use wildcard quey for donor_id and submitter_donor_id partial match
+      const regex = `*${filter.values[0].toLowerCase()}*`;
```

Comment:
Add wilecard query to partially match both donor id and submitter donor id. Converting filter value to lower case is important because donor index is using a lower case tokenizer.